lib/internal/url.js

```diff
@@ -108,6 +113,28 @@ const special = Symbol('special');
 const searchParams = Symbol('query');
 const kFormat = Symbol('format');
 
+let blob;
+let cryptoRandom;
+let threadId;
+
+function lazyBlob() {
+  if (blob === undefined)
+    blob = require('internal/blob');
+  return blob;
+}
+
+function lazyCryptoRandom() {
+  if (cryptoRandom === undefined)
+    cryptoRandom = require('internal/crypto/random');
+  return cryptoRandom;
```

Comment:
nit
```suggestion
  return cryptoRandom ??= require('internal/crypto/random');
```

Reply:
Is this going to work in `--without-ssl` builds?